recipes/jupyter-collaboration/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "jupyter-collaboration" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_collaboration-{{ version }}.tar.gz
+  sha256: 7a79651d17898564fe13ad4c31bde50059e7087346339045159126e64dd565bd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+    - hatchling >=1.4.0
+    - hatch-nodejs-version
+    - hatch-jupyter-builder
+    - jupyterlab >=4.0.0
+  run:
+    - python >=3.8
+    - jupyter_server >=2.0.0,<3.0.0
+    - jupyter_ydoc >=1.0.1,<2.0.0
+    - ypy-websocket >=0.8.3,<0.9.0
+    - jupyter_events
+    - jupyter_server_fileid >=0.6.0,<1
+
+test:
+  imports:
+    - jupyter_collaboration
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/jupyterlab/jupyter_collaboration
+  summary: 'A Jupyter extension providing support for collaborative features in JupyterLab'
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  dev_url: https://github.com/jupyterlab/jupyter_collaboration
```

Comment:
Would add:

`docs_url: https://jupyterlab-realtime-collaboration.readthedocs.io`

(that's a mouthful)

Reply:
Also might be worth a more extension, multi-line `description`.